ui-mockups/src/articles.html

```diff
@@ -0,0 +1,172 @@
+                                </small>
+                            </p>
+                            <small>
+                                <a class="ah-author-link" href="#">Meshack Mbuvi</a>
+                            </small>
+                            <p class="d-flex justify-content-between">
+                                <small class="text-muted">July 11.</small>
+                                <small class="text-muted">7 min read</small>
+                            </p>
+                        </div>
+
+                    </div>
+                </div>
+                <div class="list-group">
+                    <div class=" d-flex justify-content-between ">
+                        <img src="assets/thought-catalog-575829-unsplash.jpg" class="rounded float-left" alt="post image"
+                            width="120" height="120">
+                        <div class="ml-2">
+                            <h6><a href="article.html">Donec id elit non mi porta</a></h6>
+                            <p>
+                                <small class="text-muted ah-list-content">
+                                    Donec id elit non mi porta gravida at eget metus. Maecenas sed diam eget risus
+                                    varius blandit.
+                                </small>
+                            </p>
+                            <small>
+                                <a class="ah-author-link" href="#">Meshack Mbuvi</a>
+                            </small>
+                            <p class="d-flex justify-content-between">
+                                <small class="text-muted">July 11.</small>
+                                <small class="text-muted">7 min read</small>
+                            </p>
+                        </div>
+
+                    </div>
+                </div>
+            </div>
+            <div class="col-sm-4">
+                <div class="card">
+                    <img class="card-img-top" src="assets/alisa-anton-699811-unsplash.jpg" alt=" Card image cap">
+                    <div class="card-body">
+                        <h5 class="card-title">Card title</h5>
+                        <p class="card-text">
+                            Some quick example text to build on the card title and make up the bulk of
+                            the card's content.
+                        </p>
+                        <small>
+                            <a class="ah-author-link" href="#">Meshack Mbuvi</a>
+                        </small>
+                        <p class="card-text d-flex justify-content-between">
+                            <small class="text-muted">July 11.</small>
+                            <small class="text-muted">7 min read</small>
+                        </p>
+                    </div>
+                </div>
+            </div>
+        </div>
+        <hr>
+        <div class="jumbotron ah-jumbotron">
```

Comment:
@meshack-mbuvi consider removing this jumbotron. We do not need it here since the user has already registered and logged in
Instead of the jumbotron, use the last article layout that is on the index.html

Reply:
We also don't need an article there. I think we should have something like a footer with links to `about us, contact us, etc` pages